airbyte-integrations/connectors/source-google-search-console/acceptance-test-config.yml

```diff
@@ -38,11 +44,3 @@ acceptance_tests:
         timeout_seconds: 1800
         future_state:
           future_state_path: "integration_tests/abnormal_state.json"
-        cursor_paths:
```

Comment:
is this change intentional? what's the reason?

Reply:
No. Cursor fields already present in `configured_catalog_incremental.json`.

Reply:
Those are different. Here we should describe a path to a cursor value in a state, not in a record. Like [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-search-console/integration_tests/abnormal_state.json) and [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-search-console/source_google_search_console/streams.py#L240). If this change fixes the build, I'm afraid it is false positive

Reply:
I'm not sure because cursors for incremental tests generated here - https://github.com/airbytehq/airbyte/blob/787b2c33d7586c9a3401dd72e4fdbc2c4ccbf80c/airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py#L61

Reply:
That's right. It falls back to default value (what is in `configured_catalog_incremental.json`) if you dont set it in the acceptance-test-config.yml manually